package.json

```diff
@@ -19,30 +19,37 @@
     "testing",
     "selenium"
   ],
-  "author": "Netconomy",
+  "author": {
+    "name": "Netconomy"
+  },
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/netconomy/webdriverio-mocha-helper/issues"
   },
   "homepage": "https://github.com/netconomy/webdriverio-mocha-helper#readme",
   "devDependencies": {
-    "babel-cli": "^6.3.17",
-    "babel-eslint": "^5.0.0-beta6",
+    "async": "^1.4.0",
+    "babel-cli": "^6.7.7",
+    "babel-eslint": "^6.0.2",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-plugin-transform-object-rest-spread": "^6.3.13",
-    "babel-plugin-transform-strict-mode": "^6.3.13",
-    "babel-preset-es2015": "^6.3.13",
-    "babel-register": "^6.3.13",
-    "chai": "^3.4.1",
-    "chai-as-promised": "^5.2.0",
-    "chalk": "^1.1.1",
-    "eslint": "^1.10.3",
+    "babel-plugin-transform-object-rest-spread": "^6.6.5",
+    "babel-plugin-transform-strict-mode": "^6.6.5",
+    "babel-preset-es2015": "^6.6.0",
+    "babel-register": "^6.6.5",
+    "chai": "^3.5.0",
+    "chai-as-promised": "^5.3.0",
+    "cli-color": "^1.0.0",
```

Comment:
cli-color is used in the selenium-webdriver-testing repo. thought it might be a good idea to use just 1 logger. i can revert